src/types/sponsorPppAdjuster.ts

```
@@ -0,0 +1,29 @@
+  adjustedPrice: number;
+  adjustmentRatio: string;
+  explanationText: string;
+}
+
+export interface PppPricingResult {
+  originalPrice: number;
+  pppAdjustedPrice: number;
+  localCurrencyPrice: number;
+  pppMultiplier: number;
+  savingsPercentage: number;
+}
```

ESLint

[error] Insert `⏎`

(prettier/prettier)